services/server/src/lib/s3.ts

```diff
@@ -5,10 +5,16 @@ import {
 } from "@aws-sdk/client-s3";
 import { getSignedUrl } from "@aws-sdk/s3-request-presigner";
 
-const bucket = "litespace-assets-staging";
+const bucket = "litespace-assets";
 
 const s3 = new S3Client({
-  region: "me-south-1",
+  endpoint: "https://fra1.digitaloceanspaces.com",
+  forcePathStyle: false, // Configures to use subdomain/virtual calling format.
+  region: "fra1",
+  credentials: {
+    accessKeyId: process.env.SPACES_ACCESS_KEY || "",
+    secretAccessKey: process.env.SPACES_SECRET || "",
```

Comment:
The problem with this apporach you will not get any error if these environment variables are missing util you hit production. 

We don't access the environment variables directly. We added them to the `constants.ts` file, add basic validation and export them. Take a look at the [`emailConfig`](https://github.com/litespace-org/litespace/blob/dd8d17ccfc3b17546f72518e72133a275afb8807/services/server/src/constants.ts#L88-L96). Use the same approach to create another config and call it `spacesConfig`

Reply:
```suggestion
    accessKeyId: spaceConfig.accessKeyId,
    secretAccessKey: spaceConfig.secretAccessKey,
```